package.json

```diff
@@ -36,10 +36,10 @@
   ],
   "scripts": {
     "changelog": "conventional-changelog -p angular -u",
-    "prebuild": "rimraf ./dist && node ./scripts/generate-icon-exports.js && node ./scripts/generate-custom-properties.js",
+    "prebuild": "rimraf ./dist && node ./scripts/generate-icon-exports.js && node ./scripts/generate-custom-properties.js && tsc -d --emitDeclarationOnly --allowJs false --declarationDir dist",
     "prestorybook:start": "node ./scripts/generate-icon-exports.js && node ./scripts/generate-custom-properties.js",
     "storybook:build": "build-storybook -o .public -s docs/assets",
-    "build": "rimraf dist && cross-env NODE_ENV=production rollup --config",
+    "build": "rimraf dist && cross-env NODE_ENV=production tsc && NODE_ENV=production rollup --config",
```

Comment:
Feels a bit odd to tsc and then rollup. Can we use 
https://github.com/rollup/rollup-plugin-typescript or was there a decision against it?

Reply:
This is - apart from being a freakishly useless pkg - a good example of using rollup with a ts plugin: 
https://github.com/Rich-Harris/port-authority. On the other hand formik does was we do here with tsc and then rollup. I wonder what the reasons are. Wihout knowing any against I like a rollup plugin more 👀